### PR TITLE
Add missing french translations to netlify-cms-locales package

### PR DESCRIPTION
**Pull request by dimitrinicolas**
**Summary**

I added some missing french translations into the `netlify-cms-locales` package.

**Test plan**

/

### CODE REVIEW

File: packages/netlify-cms-locales/src/fr/index.js [hunk 1]
```diff
@@ -135,16 +145,32 @@ const fr = {
...
+        bulletedList: 'Liste à puces',
+        numberedList: 'Liste numérotée',
+        addComponent: 'Ajouter un composant',
         richText: 'Texte enrichi',
         markdown: 'Markdown',
       },
       image: {
         choose: 'Choisissez une image',
+        chooseUrl: 'Insérer depuis une adresse web',
+        replaceUrl: 'Remplacer avec une adresse web',
```
Comment (vnourdin) [inline packages/netlify-cms-locales/src/fr/index.js:164]: To be consistent with `chooseUrl`
```suggestion
        replaceUrl: 'Remplacer depuis une adresse web',
```
Same on line 172

File: packages/netlify-cms-locales/src/fr/index.js [hunk 2]
```diff
@@ -159,19 +159,19 @@ const fr = {
         markdown: 'Markdown',
       },
       image: {
-        choose: 'Choisissez une image',
+        choose: 'Choisir une image',
         chooseUrl: 'Insérer depuis une adresse web',
-        replaceUrl: 'Remplacer avec une adresse web',
+        replaceUrl: 'Remplacer depuis une adresse web',
         promptUrl: "Entrez l'adresse web de l'image",
```
Comment (vnourdin) [inline packages/netlify-cms-locales/src/fr/index.js:165]:
```suggestion
        promptUrl: "Entrer l'adresse web de l'image",
```

File: packages/netlify-cms-locales/src/fr/index.js [hunk 3]
```diff
@@ -159,19 +159,19 @@ const fr = {
...
+        chooseDifferent: 'Choisir une image différente',
         remove: "Supprimer l'image",
       },
       file: {
-        choose: 'Choisissez un fichier',
+        choose: 'Choisir un fichier',
         chooseUrl: 'Insérer depuis une adresse web',
-        replaceUrl: 'Remplacer avec une adresse web',
+        replaceUrl: 'Remplacer depuis une adresse web',
         promptUrl: "Entrez l'adresse web du fichier",
```
Comment (vnourdin) [inline packages/netlify-cms-locales/src/fr/index.js:173]:
```suggestion
        promptUrl: "Entrer l'adresse web du fichier",
```

File: packages/netlify-cms-locales/src/fr/index.js [hunk 4]
```diff
@@ -135,17 +145,33 @@ const fr = {
     },
     editorWidgets: {
       markdown: {
+        bold: 'Gras',
+        italic: 'Italique',
+        code: 'Code',
+        link: 'Lien',
+        linkPrompt: "Entrez l'adresse web du lien",
```
Comment (vnourdin) [inline packages/netlify-cms-locales/src/fr/index.js:152]:
```suggestion
        linkPrompt: "Entrer l'adresse web du lien",
```

**Review by vnourdin — COMMENTED** [review]
I think the `package-lock.json` shouldn't be there !

**Review by vnourdin — COMMENTED** [review]
Thanks for your changes :ok_hand: 
I noticed the 2 preview buttons have also missing translation: `Toggle preview` and `Sync scrolling`, if you can fix them too, it would be great!

**Review by erezrokah — APPROVED** [review]

**Comment by erezrokah** [comment]
> `Toggle preview` and `Sync scrolling`, if you can fix them too, it would be great!

I'm good with merging this as it is already an improvement. We can have another contribution for those.
Thanks @dimitrinicolas and @vnourdin!

**Comment by dimitrinicolas** [comment]
Thanks @vnourdin and @erezrokah , I will definitely take some time to internationalize the other missing sentences that are not in the JSON files as soon as I have more time.

I wish you all a good Sunday 😄

**Comment by vnourdin** [comment]
After a quick search, it turns out that those two fields aren't yet in locale files, they are hard-coded [here](https://github.com/netlify/netlify-cms/blob/e481e6e296b907c4926209edbdd8bc1ca2252fe9/packages/netlify-cms-core/src/components/Editor/EditorInterface.js#L365-L382).
What is the procedure to add them, can I just replace hard-coded values by `t('editor.editorInterface.togglePreview')` and add this new locale string in every language I can?

**Comment by erezrokah** [comment]
> After a quick search, it turns out that those two fields aren't yet in locale files, they are hard-coded [here](https://github.com/netlify/netlify-cms/blob/e481e6e296b907c4926209edbdd8bc1ca2252fe9/packages/netlify-cms-core/src/components/Editor/EditorInterface.js#L365-L382).
> What is the procedure to add them, can I just replace hard-coded values by `t('editor.editorInterface.togglePreview')` and add this new locale string in every language I can?

Ooh nice one. Yes please - that's the process.

**Comment by vnourdin** [comment]
I opened the PR #4938 with previously discussed translations.

**Pull request merged**